pages/join.html

```diff
@@ -6,3 +6,22 @@
 ---
 
 <h2>Join content goes here.</h2>
+
+<div class="container-fluid">
+	<div class="flexRow join-us-container">
```

Comment:
Let's avoid bootstrap classes.

Reply:
Nice catch! I went ahead and corrected this.